vortexasdk/result_conversions.py

```diff
@@ -30,8 +39,10 @@ def create_dataframe(
     logger.debug(f"Creating DataFrame of {logger_description}")
 
     if columns is None:
-        return pd.DataFrame(data=data, columns=default_columns)
+        df = pd.DataFrame(data=data, columns=default_columns)
     elif columns == "all":
-        return pd.DataFrame(data=data)
+        df = pd.DataFrame(data=data)
     else:
-        return pd.DataFrame(data=data, columns=columns)
+        df = pd.DataFrame(data=data, columns=columns)
```

Comment:
Could simplify this slightly

```python
columns = columns or default_columns # handles None 'if' clause
if columns == 'all':
    df = pd.DataFrame(data=data)
else:
    df = pd.DataFrame(data=data, columns=columns)
```

Reply:
This is definitely fewer lines of code - but I think it's slightly more difficult to read for the user: we mutate the columns variable and then handle it in the if statement - rather than explicitly dealing with the three posibilities for the columns variable..

Reply:
You call!